src/dune/dune.py

```diff
@@ -278,11 +278,10 @@ def export_node(self, nod, path):
                         print("\t", state.name, "was stopped due to address collision.")
 
         # Get this node ready for export.
-        if not is_active:
-            self.set_active(nod)
         if not is_running:
             self.start_node(nod)
-
+        if not is_active:
+            self.set_active(nod)
```

Comment:
I swapped `set_active` and `start_node`, because there is some issue in starting a node when it is active, but not running. This was causing `test_nodes.py` to fail.

Issue for this: https://github.com/AntelopeIO/DUNE/issues/160